commands/active_test.go

```diff
@@ -0,0 +1,51 @@
+package commands
+
+import (
+	"os"
+	"testing"
+
+	"github.com/docker/machine/drivers/fakedriver"
+	"github.com/docker/machine/libmachine/host"
+	"github.com/docker/machine/libmachine/state"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	h = host.Host{
+		Name:       "foo",
+		DriverName: "fakedriver",
+		Driver: &fakedriver.Driver{
+			MockState: state.Running,
+		},
+	}
+)
+
+func TestIsActiveReturnsFalseGivenNoActiveHost(t *testing.T) {
```

Comment:
> I'm not really a fan of that style of unit test, since it introduces complexity into the test itself.

How do you mean?

>  single test per behavior means there's no ambiguity about why a test is failing. 

I don't understand.  Tests always fail when the expected outputs don't match the actual outputs.  What's ambiguous about that?

> And a failing assertion doesn't prevent further tests from being run...

A failing assertion doesn't prevent further tests from being run AFAICT.  `assert.Equal` will invoke `t.Errorf` if the case fails and doesn't prevent the rest of the test from running `t.Fatalf` would do that.